### DISCUSSION TITLE
What do you think about Bun - Javascript?

### DISCUSSION THREAD

**Discussion by john-3219** [community/community#67277] [Discussions]
### Select Topic Area

Question

### Body

nowadays, I see that Bun is becoming quite popular and growing fast. If someone has tried it, please give me a little review about it so I can get familiar with it.

**Comment by d4kw1n** [comment]
I feel that Bun is much faster than nodejs in terms of performance, but from another perspective, it is too early to judge whether it can replace NodeJS. Previously, Deno had also considered replacing NodeJS but had not yet been able to do so. But Bun will have many advantages over Deno, so that opportunity will be much larger.
You can try it to see more interesting things...
![image](https://github.com/community/community/assets/89400593/a1c80e94-3c83-4aec-9f27-41b1d4565f54)

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
